docs/cas-server-documentation/installation/Cassandra-ticket-registry.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: default
+title: CAS - Cassandra Ticket Registry
+---
+
+# Cassandra Ticket Registry
+Cassandra integration is enabled by including the following dependency in the WAR overlay:
+
+```xml
+<dependency>
+    <groupId>org.apereo.cas</groupId>
+    <artifactId>cas-server-support-cassandra-ticket-registry</artifactId>
+    <version>${cas.version}</version>
+</dependency>
+```
+
+## Configuration Considerations
+
+There are four core configuration concerns with Cassandra:
+
+1. Keyspace
+2. Ticket cleaning
+3. Object serialization mechanism
+4. Multi datacenter replication
+
+### Keyspace
+We suggest the following Keysapace definition:
+
+- String serialized ticket:
+```cql
+CREATE KEYSPACE cas WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': '3'}  AND durable_writes = true;
```

Comment:
Do not create schemas manually as this does not scale. You want to auto-create the schema based on what you find in the catalog.